skyportal/models/user_notification.py

```diff
@@ -837,3 +862,14 @@ def receive_after_flush(session, context):
                                         url=f"/source/{target.obj_id}",
                                     )
                                 )
+
+
+def users_on_shift(session):
+    users = session.scalars(
+        sa.select(ShiftUser).where(
+            ShiftUser.shift_id == Shift.id,
+            Shift.start_date <= datetime.datetime.now(),
+            Shift.end_date >= datetime.datetime.now(),
```

Comment:
Is this where with comma seperated arguments performing an AND or OR operation? If this is an OR, it will notify users even if their shift hasn't started. I'm not sure so I prefer to ask.